interface/main/calendar/modules/PostCalendar/pntemplates/default/views/header.html

```diff
@@ -30,5 +30,5 @@
 
 </head>
```

Comment:
Brought this script into main codebase (PR is at https://github.com/openemr/openemr/pull/1129)